gpytorch/metrics/metrics.py

```diff
@@ -0,0 +1,73 @@
+from math import pi
+
+import torch
+
+from ..distributions import MultivariateNormal
+
+pi = torch.tensor(pi)
+
+
+def negative_log_predictive_density(
+    pred_dist: MultivariateNormal,
+    test_y: torch.Tensor,
+):
+
+    with torch.no_grad():
+        return -pred_dist.log_prob(test_y).item()
+
+
+def mean_standardized_log_loss(
+    pred_dist: MultivariateNormal,
+    test_y: torch.Tensor,
+):
+    """
+    Mean Standardized Log Loss.
+
+    Reference: Page No. 23,
+    Gaussian Processes for Machine Learning,
+    Carl Edward Rasmussen and Christopher K. I. Williams,
+    The MIT Press, 2006. ISBN 0-262-18253-X
+    """
+
+    with torch.no_grad():
+        f_mean = pred_dist.mean
+        f_var = pred_dist.variance
+        return 0.5 * (torch.log(2 * pi * f_var) + torch.square(test_y - f_mean) / (2 * f_var)).mean().item()
+
+
+def percentile_coverage_error(
```

Comment:
Change percentiles to quantiles as that's more correct, but this looks good.

Reply:
Done.